<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="localBox">
       <property name="text">
        <string>使用本地搜索</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="titleBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>标题</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="desBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>内容</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="authorBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>作者</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="tagsBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>tags</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="categoryBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>分类</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortKey">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>更新时间</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>创建时间</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>爱心数</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>观看数</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>章节数</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图片数</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>降序</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>升序</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>本地数量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numsLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>更新时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timesLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CompleteLineEdit" name="searchEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {background-color:transparent;}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>新到旧</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>旧到新</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最多爱心</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>最多绅士指数</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>页：0/0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="jumpPage">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>跳转</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="bookLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>屏蔽：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QtCategoryList" name="categoryList">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {background-color:transparent;}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QtBookList" name="bookList">
       <property name="styleSheet">
        <string notr="true">QListWidget {background-color:transparent;}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>大家都在搜：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QtCategoryList" name="keywordList">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget {background-color:transparent;}
QListWidget::item {
    background-color:rgb(251, 239, 243);
    color: rgb(196, 95, 125);
	border:2px solid red;
	border-color:rgb(196, 95, 125);
	border-radius: 15px;
}
/* 鼠标在按钮上时，按钮颜色 */
 QListWidget::item:hover 
{
    background-color:rgb(21, 85, 154);
    border-radius: 15px;
    color: rgb(0, 0, 0);
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtBookList</class>
   <extends>QListWidget</extends>
   <header location="global">.qtlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtCategoryList</class>
   <extends>QListWidget</extends>
   <header location="global">.qtlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CompleteLineEdit</class>
   <extends>QLineEdit</extends>
   <header>.completelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>Search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jumpPage</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>JumpPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>search</receiver>
   <slot>ChangeSort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localBox</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>SetSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortKey</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>search</receiver>
   <slot>ChangeSort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortId</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>search</receiver>
   <slot>ChangeSort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>662</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Search()</slot>
  <slot>JumpPage()</slot>
  <slot>ChangeSort()</slot>
  <slot>SetSearch()</slot>
 </slots>
</ui>
